Extensions/tweaks.js

```diff
@@ -459,7 +459,7 @@ XKit.extensions.tweaks = new Object({
 		}
 
 		if (XKit.extensions.tweaks.preferences.hide_recommended.value) {
-			XKit.extensions.tweaks.add_css("#recommended_tumblelogs, .recommended_tumblelogs, .trending_tumblelogs, .is_recommended, .recommended-unit-container { display: !important none; }", "xkit_tweaks_hide_recommended");
+			XKit.extensions.tweaks.add_css("#recommended_tumblelogs, .recommended_tumblelogs, .trending_tumblelogs, .is_recommended, .is_recommended, .recommended-unit-container { display: !important none; }", "xkit_tweaks_hide_recommended");
```

Comment:
I think the CSS here be `display: none !important;` because `display: !important none` is invalid.

![Screenshot of CSS with display: !important none;](https://i.imgur.com/ncIvjOg.png)